### DISCUSSION TITLE
Update the app icon?

### DISCUSSION THREAD

**Discussion by radical4ever** [amit9838/mousam#17] [Polls]
hi, so ik the icon is good but it could be possible to make one that match the adwaita style? , its not necessary that the icon should be completly change but just re-design it to match the libadwaita style

**Comment by amit9838** [comment]
Haven't thought about that,  may we can change it if we find any better replacement.  You are welcome if have idea regarding this.

**Reply by amit9838** [reply]
Sorry for the previous comment btw, actually I misunderstood it 😬.

**Reply by radical4ever** [reply]
didnt saw it so, dont worry 🤣

**Reply by amit9838** [reply]
would be nice to have this kind of icon, right? 
https://flathub.org/apps/org.cockpit_project.CockpitClient

**Reply by radical4ever** [reply]
> would be nice to have this kind of icon, right? https://flathub.org/apps/org.cockpit_project.CockpitClient

i would keep the circle and in the middle i'll put a sun, like the simbolic one but colored

**Reply by radical4ever** [reply]
> i would keep the circle and in the middle i'll put a sun, like the simbolic one but colored

![dev salaniLeo forecast](https://github.com/amit9838/weather/assets/83252104/2c663dfc-a8fd-4a6f-873a-87371a7aa633)

maybe like this but with a sun, and the background could have the blur from libgweather, or the gradient that weather puts when its a sunny day. i think that would look very good!

![libgweather-logo-256x256-upscaled](https://github.com/amit9838/weather/assets/83252104/285c205c-b4a1-44cc-98a7-b2f55a0c2aa2)

**Reply by amit9838** [reply]
Seems good, and lets have one version in consideration having rectangular background like the current one.

**Reply by amit9838** [reply]
May be swap this bright color with shuttle orange and push the horizon little down or ,use the current icons gradient

**Reply by radical4ever** [reply]
it could be a gradience resembling a sunset, but ok, lets keep the square icon

**Reply by radical4ever** [reply]
ok, so i've made some gradients to see what suits best 
![Captura desde 2023-06-27 19-01-03](https://github.com/amit9838/weather/assets/83252104/0f5129de-803e-4c54-9aa5-84b8027cc741)

i personally like this one and maybe with a sun in the middle it could be a really good result
![Captura desde 2023-06-27 18-59-44](https://github.com/amit9838/weather/assets/83252104/800f42e2-1a29-4319-9c64-39772b7077f2)

**Reply by amit9838** [reply]
What's your opinion about the gradient colours from current version of icon?

**Reply by amit9838** [reply]
I like that because of some reasons like-
- Difference in the color contrast among the colours being used in gradient is very shuttle
- It matches with clear sky theme
- Icons above the gradient have high difference in contrast wth the background.
- simple cloud and sun.

**Reply by amit9838** [reply]
I was thinking it would be great if we add shuttle shadow on icons like the above (cockpit) and then use adwaita guidelines to match the perspective.

**Reply by radical4ever** [reply]
> What's your opinion about the gradient colours from current version of icon?

They're good too that's an option i completely forgot 😅🤣

**Reply by radical4ever** [reply]
> I was thinking it would be great if we add shuttle shadow on icons like the above (cockpit) and then use adwaita guidelines to match the perspective.

It could be a good start, then its just a matter of time to polish the icon follow libadawaita guidelines, and maybe on a future be part of gnome circle

**Reply by radical4ever** [reply]
I think that just taking this icon and just give it the libadwaita touch is enough, i mean, its the original icon and change it would mean like... swapping its face 😅. It would be like a makeover🤣

**Reply by amit9838** [reply]
Yes exactly 😄.
